package.json

```diff
@@ -47,6 +47,7 @@
     "zod": "^3.8.2"
   },
   "devDependencies": {
+    "@types/qrcode.react": "^1.0.2",
```

Comment:
This dependency should be in the `dependencies` object instead of the `devDependencies` one.